data.json

```
@@ -89,6 +89,15 @@
     "dateAdded": "2019-09-25T17:27:00.000Z",
     "tags": ["clothing", "hacktoberfest"]
   },
+  {
+    "name": "CircleCI Hacktoberfest 2019",
+    "difficulty": "medium",
+    "description": "Participate in #Orbtoberfest with CircleCI and claim limited edition swag! 1 - 3 pull requests: Limited-Edition Sticker, 4+ pull requests: Limited-Edition T-shirt",
+    "reference": "https://hacktoberfest.circleci.com/#/" ,
+    "image": "https://hacktoberfest.circleci.com/img/hacktober19_swag-mockup.5c199128.png",
+    "dateAdded": "2019-09-29T19:30:34+0200",
```

```suggestion
    "dateAdded": "2019-09-29T19:30:00.000Z",
```

This should fix tests